scenarios/remote_config/test_remote_configuration.py

```diff
@@ -118,6 +118,7 @@ class Test_RemoteConfigurationUpdateSequenceFeatures(RemoteConfigurationFieldsBa
     request_number = 0
 
     @bug(context.weblog_variant == "spring-boot-openliberty", reason="APPSEC-6721")
+    @bug(context.library >= "java@0.116.0", reason="Failing test")
```

Comment:
This seems to fail in nightlies because they inject a custom rules file, which inhibit remote config in Java. But it's ok to mark this as bug until we align this behavior.